src/main/java/reactor/netty/transport/ProxyProvider.java

```diff
@@ -245,7 +233,7 @@ private String getPasswordValue() {
 		String host;
 		int port;
 		Supplier<? extends InetSocketAddress> address;
-		String nonProxyHosts;
+		Predicate<SocketAddress> shouldProxy = a -> true;
```

Comment:
I found myself confused a couple of times handling the not notProxied double negative. Since we eventually call this in the ```shouldProxy()``` method I figured I could stick with that. But since this is related to specifically nonProxyHosts, I am happy to bring it back.